web/app/view/ReportConfigController.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2016 Anton Tananaev (anton.tananaev@gmail.com)
+ * Copyright 2016 Andrey Kunitsyn (abyss@fox5.ru)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+Ext.define('Traccar.view.ReportConfigController', {
+    extend: 'Ext.app.ViewController',
+    alias: 'controller.reportConfigDialog',
+
+    init: function () {
+        var store = this.lookupReference('eventTypeField').getStore();
+        if (store.getCount() === 0) {
+            store.add({
+                type: Traccar.store.ReportEventTypes.allEvents,
```

Comment:
I think the rule should be that if you use some class name, you should import it. When you use `Ext.getStore`, you don't use class name, you just use store id. For example, if you use store class name explicitly anywhere, you should import it. `Traccar.Style` is the only exception because it's used everywhere. `Traccar.AttributeFormatter` is not an exception. If there is no import for it, I guess we should add it. Chained import is not enough. I think it's better to always import any classes that are used explicitly.